src/controllers/rankings.js

```diff
@@ -53,6 +53,26 @@ function depthFromCtx(ctx) {
   return pathOr(10, ['request', 'query', 'depth'], ctx)
 }
 
+async function setUserRanking(ctx, next) {
+  const UserRanking = ctx.app.models.UserRanking;
+  const graph = ctx.request.body;
+  await UserRanking.query()
+    .where({ id: graph.item_id })
+    .first()
+    .then(existingRanking => {
+      if (!existingRanking) {
+        // HACK: Attach the user ID to the graph since UserRanking.insertAfter needs it.
+        graph.user_id = ctx.state.user.id;
```

Comment:
I don't love this but it was the path of least resistance and I couldn't think of a better way short of refactoring the `insertAfter` method.  Alternate approaches welcome.